src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null
+                || password.isEmpty() || repeatPassword.isEmpty()) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
```

Comment:
let's use only one `if` check and put all the conditions there

consider to simplify your checking - do you need `isEmpty()` check if you will check the length anyway?